doc/api/path.md

```diff
@@ -48,24 +103,30 @@ process.env.PATH.split(path.delimiter)
 
 ## path.dirname(path)
 
-Return the directory name of a path, similar to the Unix `dirname` command.
-`path` must be a string.
+* `path` {String} A file path
 
-Example:
+The `path.dirname()` method returns the directory name of a `path`, similar to
+the Unix `dirname` command.
+
+For example:
 
 ```js
 path.dirname('/foo/bar/baz/asdf/quux')
 // returns '/foo/bar/baz/asdf'
 ```
 
+A [`TypeError`][] is thrown if `path` is not a string.
+
 ## path.extname(path)
 
-Return the extension of the path, from the last '.' to end of string
-in the last portion of the path.  If there is no '.' in the last portion
-of the path or the first character of it is '.', then it returns
-an empty string. `path` must be a string.
+* `path` {String} A file path
 
-Examples:
+The `path.extname()` method returns the extension of the `path`, from the last
+occurance of the `.` (period) character to end of string in the last portion of
+the `path`.  If there is no `.` in the last portion of the `path`, or if the
+first character of the `path` is `.`, then an empty string is returned.
```

Comment:
This isn't quite correct. It doesn't necessarily have to be the first character of the *entire* `path`, just as long as it's the first character in the basename (`path.basename(path)`). So for example `/foo/bar/.baz` and `.baz` would both return empty strings.